abstract.md

```diff
@@ -0,0 +1,41 @@
+# Making research software more robust
+Morgan Taschuk and Greg Wilson
+
+
+At the cutting edge of research, bioinformatics software is often written by a
```

Comment:
I would remove the opening clause - this is true everywhere, not just at the bleeding edge.